contentcuration/contentcuration/views.py

```diff
@@ -53,11 +54,11 @@ def channel(request, channel_id):
     channel = get_object_or_404(Channel, id=channel_id, deleted=False)
     channel_serializer =  ChannelSerializer(channel)
 
-    accessible_channel_list = Channel.objects.filter( Q(deleted=False, public=True) | Q(deleted=False, editors__email__contains= request.user))
+    accessible_channel_list = Channel.objects.filter( Q(deleted=False, public=True) | Q(deleted=False, editors__email__contains= request.user.email))
```

Comment:
This should also be possible to simplify down to something like:
`Channel.objects.filter(deleted=False).filter(Q(public=True) | Q(editors=request.user))`